pages/common/npm.md

```diff
@@ -1,10 +1,7 @@
 # npm
 
-> Node package manager, to manage Node.js projects and install module dependencies.
-
-- Create a new project in the current folder:
-
-`npm init`
+> JavaScript and Node.js package manager.
+> Manage Node.js projects and their module dependencies.
 
 - Download all dependencies referenced in package.json:
```

Comment:
I feel like many more people use npm just as package consumers than as package creators, hence my suggestion to have install [-g] first. But I'll defer to you guys who are more familiar with the tool to make the decisions :)